web/services/models.py

```diff
@@ -56,9 +56,13 @@ class RecoveryFiles(models.Model):
 
 
 def send_email_notification(sender, instance, signal, *args, **kwargs):
-    name = instance.user.name if instance.user else ''
-    email = instance.user.email if instance.user else ''
-    text_box = instance.textBox
+    name = instance.user.name if instance.user and \
+        instance.user.name != 'public_name' else ''
```

Comment:
Do we want to assert on everything or just the minimal, obviously fake, email address?

Reply:
There are some tests in the system test which don't provide an email address as all combinations of what may be provided are checked. Currently a notifiaction email is sent if the user, provides a name or an email or puts information in the text box so we need to check all three. 

Would it be better to only send a notification email if an email is provided.